build/charts/antrea/templates/agent/daemonset.yaml

```diff
@@ -130,8 +135,15 @@ spec:
           image: {{ include "antreaAgentImage" . | quote }}
           imagePullPolicy: {{ include "antreaAgentImagePullPolicy" . }}
           {{- if ((.Values.testing).coverage) }}
-          command: ["/bin/sh"]
-          args: ["-c", "sleep 2; antrea-agent-coverage -test.run=TestBincoverRunMain -test.coverprofile=antrea-agent.cov.out -args-file=/agent-arg-file; while true; do sleep 5 & wait $!; done"]
+          args:
```

Comment:
Where did you call utils/run_cov_binary.sh?

Reply:
It is called as `ENTRYPOINT` in the dockerfile.